packages/api/src/external/carequality/gateway/index.ts

```diff
@@ -52,6 +58,15 @@ export async function gatherXCPDGateways(patient: Patient): Promise<Gateways> {
   };
 }
 
+export function facilitiesWithEpicFilter(
+  cqDirectoryEntries: CQDirectoryEntry[],
+  isEpicEnabled: boolean
+): CQDirectoryEntry[] {
+  return isEpicEnabled
+    ? cqDirectoryEntries
+    : cqDirectoryEntries.filter(entry => entry.managingOrganization !== EPIC_ORG_NAME);
```

Comment:
When using string, I like to trim and lowercase it, you never know when someone add a string with an unexpected casing